docs/getting-started/release_package.md

```diff
@@ -0,0 +1,132 @@
+# Setup using Release package
+
+## Prerequisites
+
++ [Install docker](https://docs.docker.com/install/)
+
++ [Install kubeadm/kubectl](https://kubernetes.io/docs/setup/independent/install-kubeadm/)
+
++ [Creating cluster with kubeadm](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
+
++ After initializing Kubernetes master, we need to expose insecure port 8080 for edgecontroller/kubectl to work with http connection to Kubernetes apiserver.
+  Please follow below steps to enable http port in Kubernetes apiserver.
+
+    ```shell
+    vi /etc/kubernetes/manifests/kube-apiserver.yaml
+    # Add the following flags in spec: containers: -command section
+    - --insecure-port=8080
+    - --insecure-bind-address=0.0.0.0
+    ```
+
++ (**Optional**)KubeEdge also supports https connection to Kubernetes apiserver. Follow the steps in [Kubernetes Documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to create the kubeconfig file.
+
+  Enter the path to kubeconfig file in controller.yaml
+  ```yaml
+  controller:
+    kube:
+      ...
+      kubeconfig: "path_to_kubeconfig_file" #Enter path to kubeconfig file to enable https connection to k8s apiserver
+  ```
+  
+  ## Cloud Vm
+ 
+  **Note**:execute the below commands as root user
+  ```
```

Comment:
this should be  ```sh

Reply:
applies to all other places where we have steps to run some shell commands.

Reply:
ok